driver-core/src/test/functional/com/mongodb/client/model/expressions/ArrayExpressionsFunctionalTest.java

```diff
@@ -127,41 +127,128 @@ public void mapTest() {
                 "{'$map': {'input': [true, true, false], 'in': {'$not': '$$this'}}}");
     }
 
+    // https://www.mongodb.com/docs/manual/reference/operator/aggregation/reduce/
+    // reduce is implemented as each individual type of reduction (monoid)
+    // this prevents issues related to incorrect specification of identity values
+
     @Test
-    public void reduceTest() {
-        // https://www.mongodb.com/docs/manual/reference/operator/aggregation/reduce/
+    public void reduceAnyTest() {
```

Comment:
I propose calling all the tests correspondingly to the tested method names, i.e., without the "reduce" part.

Reply:
There is both concat(v -> v) and concat(otherArray).